src/context/AppProvider/AppContext.tsx

```diff
@@ -186,40 +185,19 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
         dispatch(getValidatorData.initiate(accountId, options))
       }
 
-      await fetchAllOpportunitiesMetadata()
+      await fetchAllOpportunitiesIds({ forceRefetch: true })
+      await fetchAllOpportunitiesMetadata({ forceRefetch: true })
 
-      requestedAccountIds.forEach(async accountId => {
+      requestedAccountIds.forEach(accountId => {
         const { chainId } = fromAccountId(accountId)
         switch (chainId) {
           case cosmosChainId:
           case osmosisChainId:
             dispatch(getValidatorData.initiate(accountId, options))
             break
           case ethChainId:
-            await dispatch(
-              opportunitiesApi.endpoints.getOpportunityIds.initiate({
-                defiType: DefiType.Staking,
-                defiProvider: DefiProvider.Idle,
-              }),
-            )
-            await dispatch(
-              opportunitiesApi.endpoints.getOpportunitiesMetadata.initiate({
-                defiType: DefiType.Staking,
-                defiProvider: DefiProvider.Idle,
-                opportunityType: DefiType.Staking,
-              }),
-            )
-            // await dispatch(
-            // opportunitiesApi.endpoints.getOpportunitiesUserData.initiate({
-            // accountId,
-            // defiType: DefiType.Staking,
-            // defiProvider: DefiProvider.Idle,
-            // opportunityType: DefiType.Staking,
-            // }),
-            // )
-
             // Don't await me, we don't want to block execution while this resolves and populates the store
-            fetchAllOpportunitiesUserData(accountId)
+            fetchAllOpportunitiesUserData(accountId, { forceRefetch: true })
```

Comment:
accountId *should* be unique enough that we don't require forceRefetch here? these effect dependencies will change a lot

Reply:
Yeah, had it as an unsafe safety net here during development as FoxEthProvider was currently rugging us, but I've removed the fetching there and it works perfectly without `forceRefetch` now, removed in all 3 thunks